<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngineRegistrationDialog</class>
 <widget class="QDialog" name="EngineRegistrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>エンジン登録</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QListWidget" name="engineListWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <widget class="QPushButton" name="fontDecreaseButton">
       <property name="text">
        <string>A-</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fontIncreaseButton">
       <property name="text">
        <string>A+</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addEngineButton">
       <property name="text">
        <string>追加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeEngineButton">
       <property name="text">
        <string>削除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="configureEngineButton">
       <property name="text">
        <string>設定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>閉じる</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
